media/css/firefox/channel.scss

```diff
@@ -83,11 +83,17 @@ $image-path: '/media/protocol/img';
 // styles from developer/includes/newsletter.scss
 
 .t-newsletter {
+    background: #08204e url("/media/img/firefox/developer/newsletter-bg.5cb53dbe3960.svg") center bottom no-repeat;
+    background-size: cover;
```

Comment:
Let's not include the developer page background and remove these lines.

Reply:
Made the required changes.
Thanks.